test/aggregation-builder.js

```diff
@@ -412,3 +412,51 @@ test('aggregationBuilder | matrix stats', (t) => {
     }
   })
 })
+
+test('aggregationBuilder | metadata', (t) => {
+    t.plan(1)
+
+    const result = aggregationBuilder()
+        .aggregation('terms', 'title', { _metadata: { color: 'blue' } }, 'titles')
+
+    t.deepEqual(result.getAggregations(), {
+        titles: {
+            terms: {
+                field: 'title'
+            },
+            meta : {
+              color : 'blue'
+            }
+        }
+    })
+})
+
+test('aggregationBuilder | nested metadata', (t) => {
+    t.plan(1)
+
+    const result = aggregationBuilder()
+        .aggregation('terms', 'title', { _metadata: { color: 'blue' } }, 'titles', (a) => {
```

Comment:
hmm, I'd suggest using `_meta` instead of `_metadata` to stay closer to the terminology ES uses. (Wouldn't use `meta` though, because bodybuilder needs to change the nesting, as you know and implemented...)